### PR TITLE
added scriptlets-1 test

### PR DESCRIPTION
**Pull request by glum23**
added a test for rpm scriptlets

### CODE REVIEW

File: dnf-docker-test/features/scriptlets-1.feature [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        And I run "dnf -y install Test-posttrans-ok"
+       Then rpmdb changes are
+         | State     | Packages          |
+         | installed | Test-posttrans-ok |
+        And the command stdout should match regexp "stdout-posttrans"
+
+  Scenario: Install a pkg with a failing %pre scriptlet
+       When I save rpmdb
+        And I run "dnf -y install Test-pre-fail"
+       Then rpmdb changes are
```
Comment (ignatenkobrain) [inline dnf-docker-test/features/scriptlets-1.feature]: there is shortcut `rpmdb doesn't change` or something like that

File: dnf-docker-test/features/scriptlets-1.feature [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  Scenario: Install a pkg with a failing %posttrans scriptlet
+       When I save rpmdb
+        And I run "dnf -y install Test-posttrans-fail"
+       Then rpmdb changes are
+         | State     | Packages            |
+         | installed | Test-posttrans-fail |
+        And the command stderr should match regexp "Non-fatal POSTTRANS scriptlet failure in rpm package Test-posttrans-fail"
+
+  Scenario: Remove a pkg with a successful %preun scriptlet
+       When I run "dnf -y install Test-preun-ok"
```
Comment (ignatenkobrain) [inline dnf-docker-test/features/scriptlets-1.feature]: I would check that it was also installed

Comment (glum23) [inline dnf-docker-test/features/scriptlets-1.feature]: OK, I'll take a look at both tomorrow.

**Comment by glum23** [comment]
updated as requested

**Pull request merged**